Px.Utils.UnitTests/Validation/DataValidationTests/DataNumberValueValidator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using PxUtils.Validation;
+using PxUtils.Validation.DataValidation;
+
+namespace Px.Utils.UnitTests.Validation.DataValidationTests;
+
+[TestClass]
+public class DataNumberValueValidatorTest
+{
+
+    [TestMethod]
+    [DataRow("0")]
+    [DataRow("0.1")]
+    [DataRow("1.1")]
+    [DataRow("-1.1")]
+    [DataRow("1234567890")]
+    [DataRow("-9999999999999999999999999999")]
+    [DataRow("9999999999999999999999999999")]
+    [DataRow("-79228162514264337593543950335")]
+    [DataRow("79228162514264337593543950335")]
+    public void AllowedNumberValues(string allowedValue)
+    {
+        var validator = new DataNumberDataValidator();
```

Comment:
Suugest using explicity variable types and simplified object initialization
`DataNumberDataValidator validator = new();`

Reply:
will change all if/when that is the desired style